### ISSUE TITLE
Focus highlights lost after using native Voice Over Command

### ISSUE THREAD

**Issue by terracoda** [phetsims/ohms-law#86]
Mentioned in #74, 
> pink focus highlights go away after I use native Voice Over Commands.

By native Voice Over Commands, I mean VO shortcuts, like Control+Option+B to start reading from beginning of sim.

I'm going to test with a a couple of more commands.

**Comment by jessegreenberg** [comment]
I was able to reproduce this bug.  I had focus on a slider, then pressed ctrl +option + b to read from the top.  Focus disappeared after reading through the first several lines, then didn't come back.

document.activeElement is the input element that should have focus.

**Comment by jessegreenberg** [comment]
Closing this issue, it is going to be tracked in https://github.com/phetsims/a11y-research/issues/71. It does not need to block publication of ohms law, but should be investigated with high priority.

**Comment by terracoda** [comment]
Thanks fir starting the a11y-research issue, @jessegreenberg
